packages/account-v2/src/constants/manualFormConstants.tsx

```diff
@@ -6,18 +6,21 @@ import NIMCSlipFront from '../assets/manual-upload/nimc-slip-front.svg';
 import PassportPlaceholder from '../assets/manual-upload/passport-placeholder.svg';
 import ProofOfAgeIcon from '../assets/manual-upload/proof-of-age.svg';
 
-export type TManualDocumentTypes = typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES];
+export type TManualDocumentTypes = Exclude<
+    typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES],
+    'birth_certificate'
+>;
 
 export const MANUAL_DOCUMENT_TYPES = Object.freeze({
```

Comment:
same suggestion for this file, lets use  'as const' instead of Object.freeze.
how do you think?

Reply:
This is single level. I think its OK